ovh/resource_cloud_project_workflow_backup_test.go

```diff
@@ -0,0 +1,121 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+const WORKFLOW_BACKUP_TEST_CONF = `
+resource "ovh_cloud_project_workflow_backup" "my_backup"{
+	service_name		= "%s"
+	region_name				= "%s"
+	cron				= "50 4 * * *"
+	instance_id			= "%s"
+	max_execution_count	= "0"
+	name				= "%s"
+	rotation			= "7"
+}
+`
+const WORKFLOW_BACKUP_TEST_CONF_UPDATED = `
+resource "ovh_cloud_project_workflow_backup" "my_backup"{
+	service_name		= "%s"
+	region_name				= "%s"
+	cron				= "50 4 * * *"
+	instance_id			= "%s"
+	max_execution_count	= "0"
+	name				= "%s"
+	rotation			= "5"
+}
+`
+
+const CLOUD_PROJECT_TEST_ENV_VAR = "OVH_CLOUD_PROJECT_SERVICE_TEST"
+const WORKFLOW_BACKUP_TEST_REGION_ENV_VAR = "OVH_CLOUD_PROJECT_WORKFLOW_BACKUP_REGION_TEST"
```

Comment:
Add `OVH_CLOUD_PROJECT_WORKFLOW_BACKUP_INSTANCE_ID_TEST` and `OVH_CLOUD_PROJECT_WORKFLOW_BACKUP_REGION_TEST` new environment variables in the home page documentation: index.html.markdown

Reply:
done